tests/McpServerTests/McpPromptRegistryTests.cs

```diff
@@ -0,0 +1,127 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using nanoFramework.TestFramework;
+using nanoFramework.WebServer.Mcp;
+
+namespace McpServerTests
+{
+    // Test prompt classes with MCP prompts
+    public static class TestPromptsClass
+    {
+        [McpServerPrompt("simple_prompt", "A simple test prompt")]
+        public static string SimplePrompt()
+        {
+            return "This is a prompt example";
+        }
+
+        [McpServerPrompt("anohter_simple_prompt", "Another test prompt")]
+        public static string AnotherSimplePrompt()
+        {
+            return "This is another prompt example";
+        }
```

Comment:
create another one with more than 1 parameter. In that case, you can check it's not part of the prompt list

Reply:
Done, even added more than that